components/dashboard/src/service/service.tsx

```diff
@@ -237,7 +237,9 @@ export class IDEFrontendService implements IDEFrontendDashboardService.IServer {
         const [user, listener, ideCredentials] = await Promise.all([
             this.service.server.getLoggedInUser(),
             this.service.listenToInstance(this.workspaceID),
```

Comment:
(off-topic) but we should replace it with `watchWorkspaceStatuses` in another PR cc @mustard-mh